packages/react-vapor/src/components/logoCard/LogoCard.tsx

```diff
@@ -91,5 +94,13 @@ export class LogoCard extends React.Component<ILogoCardProps & React.HTMLProps<H
                 {ribbon}
             </div>
         );
+
+        return this.props.disabled ? (
+            <Tooltip title={this.props.tooltip} placement={this.props.tooltipPlacement}>

```

Comment:
@rosalie-liu Crois-tu que c'est mieux de faire encore un cas générique (return logoCard avec un tooltip dans tous les cas avec `title={this.props.disabled ? this.props.tooltip : ''}`

Reply:
Hmm si ça brise pas le style, je pense pas qu'on en a besoin :thinking:

Reply:
Looking at the [tooltip implementation](https://github.com/coveo/react-vapor/blob/fcd398a8b29252804be8d569b0d05f9655c97220/packages/react-vapor/src/components/tooltip/Tooltip.tsx#L126), we can see that if the `title` is falsy it will not render anything other than the tooltip children, so we should be good indeed.

Reply:
Thanks a lot Gael